org.adempiere.base/src/org/compiere/util/Msg.java

```diff
@@ -365,6 +365,29 @@ public static String getMsg (Language language, String AD_Message)
 		return getMsg (language.getAD_Language(), AD_Message);
 	}   //  getMeg
 
+	/**
+	 * Text returned by the getMsg/translator function is usually in the format message + SEPARATOR + tip.<br/>
+	 * This function is used for separating these parts
+	 * This function help to separate it 
+	 * @param msgFull
+	 * @return
+	 */
+	public static ValueNamePair getMsgInfo (String msgFull) {
```

Comment:
The method name is not obvious for what it does. Suggest to rename it to "splitToMsgAndTip"